pom.xml

```diff
@@ -26,7 +26,7 @@
   <groupId>org.thymeleaf</groupId>
   <artifactId>thymeleaf</artifactId>
   <packaging>jar</packaging>
-  <version>3.0.14-SNAPSHOT</version>
+  <version>3.1.0-SNAPSHOT</version>
```

Comment:
`3.0.14.RELEASE` has just been published, and I've created a new `3.1-dev` branch meant for development of 3.1.x, starting from `3.0.15-SNAPSHOT`. Could this be rebased on that new branch, please?

Reply:
Sure thing. I've rebased and updated the PR to point to 3.1-dev